mlir/include/mlir/Target/LLVMIR/ModuleImport.h

```diff
@@ -281,6 +282,9 @@ class ModuleImport {
   /// after the function conversion has finished.
   void addDebugIntrinsic(llvm::CallInst *intrinsic);
 
+  /// Similar to `addDebugIntrinsic`, but for debug records.
+  void addDebugRecord(llvm::DbgRecord *dr);
```

Comment:
```suggestion
  /// Adds a debug record to the list of records that have to be converted
  /// after the function conversion finished.
  void addDebugRecord(llvm::DbgRecord *debugRecord);
```
nit: I would spell out the comment and use a longer variable name.